src/common/icons/icon.scss

```diff
@@ -76,3 +76,28 @@
         @include getCheckContainerStyle($iconSize, $border-size);
     }
 }
+
+@mixin inapplicable-icon-styles($iconSize, $border-size: 1px, $iconColor: $neutral-0) {
+    $true-icon-size: ($iconSize - $border-size * 2);
+    $width-value: (1/7 * $true-icon-size);
```

Comment:
can you add comments for the logic behind `1/7` here and `4/7` down below? I found some other ones on the cross icon (`2/14` and `8/14`) and I have no idea how did we come with those numbers, it would be nice to have a comment to explain it... or maybe a couple of well-named variables for numerator and denominator

Reply:
@devaradhanm Do you still remember how did you figure out these numbers?